addons/website/migrations/11.0.1.0/post-migration.py

```diff
@@ -15,21 +15,20 @@ def fill_website_pages(env):
         INSERT INTO website_page
             (url, view_id, website_indexed, website_published, create_uid,
              create_date, write_uid, write_date)
-        SELECT
-            wm.url, iuv.id, TRUE, TRUE, iuv.create_uid,
-            iuv.create_date, iuv.write_uid, iuv.write_date
-        FROM
-            ir_ui_view iuv
-        INNER JOIN
-            website_menu wm ON wm.url = '/page/' || iuv.name AND
-            (wm.website_id = iuv.website_id OR
-             wm.website_id IS NULL OR
-             iuv.website_id IS NULL)
-        LEFT JOIN
-            website_page wp ON wp.view_id = iuv.id
-        WHERE
-            wp.id IS NULL AND
-            iuv.page = true""",
+	SELECT COALESCE(wm.url, '/page/' || iuv.name), iuv.id, TRUE, TRUE, iuv.create_uid,
+		iuv.create_date, iuv.write_uid, iuv.write_date
+	FROM
+	    website_page wp
+	RIGHT JOIN
```

Comment:
Why `RIGHT JOIN`? It should be `LEFT JOIN` as well

Reply:
I doubt that @pedrobaeza .. LEFT JOIN would just fetch all records of website_page table whereas we need to fetch all records of ir_ui_view table inrespective of them satisfying the join condition of not so RIGHT JOIN seems more appropriate.

Also LEFT JOIN doesn't work ;)

Reply:
OK, I get you. I thought all views had `website.page` record.